vllm/executor/ray_gpu_executor.py

```diff
@@ -292,8 +308,7 @@ def _run_workers(
         self,
         method: str,
         *args,
-        driver_args: Optional[List[Any]] = None,
-        driver_kwargs: Optional[Dict[str, Any]] = None,
+        async_remote_only: bool = False,
```

Comment:
A slightly more accurate name:
```suggestion
        remote_worker_only: bool = False,
```

Reply:
@zhuohan123 I've renamed it for now to `remote_workers_only_async` because in addition to only running on the remote workers, it also changes the behavior to not wait for the responses and return the future(s) rather than resulting outputs.

Let me know if you would still prefer to drop the `_async` though (or if you can think of a better name... maybe `start_remote_workers_only`?) and I'll update again.